src/ui/components/DashKit/plugins/Control/Control.tsx

```diff
@@ -1016,12 +1029,21 @@ class Control extends React.PureComponent<PluginControlProps, PluginControlState
 
                 const valueWithOperation = addOperationForValue({
                     value,
-                    operation: source.operation,
+                    operation,
                 });
 
                 this.onChange(param, valueWithOperation);
             };
 
+            const label = showTitle ? getRequiredLabel({title, required}) : '';
```

Comment:
@Marginy605 
I suggest to make it somethig like this:

```js
let label = '';
let innerLabel = '';

if (showTitle) {
    // if showing title than don't add possible asterisk to innerLabel
    label = getRequiredLabel({title, required});
    innerLabel = innerInnerTitle ? (innerTitle || '') : '';
} else {
    // if not showing title that trying to add asterisk to innerLabel
    label = '';
    innerLabel = showInnerTitle ? getRequiredLabel({title: innerTitle, required}) : '';
}
```
And remove `getRequiredInnerLabel`